Dockerfile

```diff
@@ -14,5 +14,8 @@ WORKDIR /pwru
 COPY . .
 RUN make
 RUN chmod a+x /pwru
-ENTRYPOINT ["./pwru"]
-CMD ["./pwru", "-output-tuple"]
+
+FROM ubuntu:20.04 AS base
+COPY --from=build /pwru/pwru /usr/local/bin/
+ENTRYPOINT ["/usr/local/bin/pwru"]
+CMD ["/usr/local/bin/pwru", "-output-tuple"]
```

Comment:
Let's drop `CMD`. AFAIK, this then would just run `pwru`.

Reply:
Yes dropping CMD would just run the ENTRYPOINT. Have removed CMD.